src/ship/controller/physicaldevice/ConnectedPhysicalDeviceManager.cpp

```diff
@@ -52,6 +55,13 @@ void ConnectedPhysicalDeviceManager::RefreshConnectedSDLGamepads() {
     mConnectedSDLGamepads.clear();
     mConnectedSDLGamepadNames.clear();
 
+#ifdef ENABLE_EXP_AUTO_CONFIGURE_CONTROLLERS
+    for (uint8_t i = 0; i < 4; i++) {
+        mIgnoredInstanceIds[i].clear();
+    }
+#endif
+
+    uint8_t port = 0;
```

Comment:
```suggestion
#ifdef ENABLE_EXP_AUTO_CONFIGURE_CONTROLLERS
    for (uint8_t i = 0; i < 4; i++) {
        mIgnoredInstanceIds[i].clear();
    }
    
    uint8_t port = 0;
#endif
```

switching from using `port` to `j` lower down makes the existing logic harder to understand. this makes it so we can keep using `port` there

Reply:
same as above